xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVTBGLES.h

```diff
@@ -39,7 +39,6 @@ class CRendererVTB : public CLinuxRendererGLES
 
   // Feature support
   virtual bool Supports(EINTERLACEMETHOD method) override;
-  virtual EINTERLACEMETHOD AutoInterlaceMethod() override;
```

Comment:
the supports method needs to be removed here too i guess ... at least my compiler tells me that supports method is marked as override baut there is nothing in the base class to be overridden...